.github/PULL_REQUEST_TEMPLATE/HOTFIX.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Hotfix YYYY-MM-DD
+about: When applying a hotfix for the example site
+title: Hotfix YYYY-MM-DD
+labels: hotfix
+assignees: ''
+
+---
+
+- [ ] Start a new hotfix branch:
+```bash
+$ git flow hotfix start YYYY-MM-DD
+```
+- [ ] Make commits to hotfix branch to address any issues that need to be resolved
+- [ ] Publish the hotfix branch:
+```bash
+$ git flow hotfix publish YYYY-MM-DD
+```
+- [ ] Create a PR for the hotfix branch, target it to `main`, have it reviewed, and confirm that all checks pass
+- [ ] Finish and publish the hotfix branch:
+    - When prompted, keep default commit messages
+    - Use `-n` to skip tagging
+    - Use `-p` to push changes to the remote repository
+```bash
+$ git flow hotfix finish -n -p
```

Comment:
#27 was squashed with the `-S` option in an attempt to get it more like our typical feature branch merges, but in hindsight I don't love the default commit message formatting and think it would have been more valuable to preserve the individual commits. I removed the `-S` option from the `hotfix finish` command as a result.